system_test/common/aest_db_SUITE.erl

```diff
@@ -95,9 +95,21 @@ format(Fmt, Args) ->
 
 node_spec(Name, DbHostPath) ->
     DbGuestPath = "/home/aeternity/node/data/mnesia",
-    aest_nodes:spec(Name, [], #{source  => {pull, "aeternity/aeternity:local"}, db_path => {DbHostPath, DbGuestPath}}).
+    aest_nodes:spec(Name, [], #{source  => {pull, "aeternity/aeternity:local"},
+                                db_path => {DbHostPath, DbGuestPath},
+                                genesis_accounts => genesis_accounts()}).
```

Comment:
Could you please enrich type [`aest_docker:node_spec`](https://github.com/aeternity/aeternity/blob/831e1ee30e4092058ca35573bdfd8dd5f392c01d/system_test/common/helpers/aest_docker.erl#L61) with any genesis-related features (optional `=>` field plus comment).

Reply:
Addressed in commit [31e601c](https://github.com/aeternity/aeternity/pull/2214/commits/31e601c2a23cebf18201bf08018e324d5849d49a)